<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sample scan options</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QFrame" name="dirNameFrame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="browseBtn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="dirPathEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Scan options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="1">
         <widget class="QComboBox" name="scanModeCombo">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>All files (slower)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>By extension</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Scan mode:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QListView" name="formatList">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Formats:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="allFormatsChk">
          <property name="text">
           <string>All</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Channels</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QComboBox" name="channelsCombo">
          <item>
           <property name="text">
            <string>Mono/Stereo</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Mono</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Stereo</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QListView" name="sampleRatesList"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Sample rates:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="allSampleRatesChk">
          <property name="text">
           <string>Any</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_6">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QCheckBox" name="sizeBiggerChk">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Bigger than</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QCheckBox" name="sizeSmallerChk">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Smaller than</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="lengthLongerSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <number>3600</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QCheckBox" name="lengthLongerChk">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Longer than</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5">
         <widget class="QComboBox" name="sizeSmallerCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>MB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>KB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QCheckBox" name="lengthShorterChk">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Shorter than</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QComboBox" name="lengthLongerCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>minutes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>seconds</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QSpinBox" name="lengthShorterSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>3600</number>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QComboBox" name="lengthShorterCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <item>
           <property name="text">
            <string>minutes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>seconds</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="sizeBiggerCombo">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>MB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>KB</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>B</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="1" column="0" colspan="6">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QDoubleSpinBox" name="sizeSmallerSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <double>0.010000000000000</double>
          </property>
          <property name="maximum">
           <double>1024.000000000000000</double>
          </property>
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QDoubleSpinBox" name="sizeBiggerSpin">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="maximum">
           <double>1024.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeBiggerChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>sizeBiggerCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>58</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeSmallerChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>sizeSmallerCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>332</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthLongerChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>lengthLongerSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthLongerChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>lengthLongerCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthShorterChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>lengthShorterSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>448</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lengthShorterChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>lengthShorterCombo</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>377</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeBiggerChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>sizeBiggerSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>146</x>
     <y>309</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sizeSmallerChk</sender>
   <signal>toggled(bool)</signal>
   <receiver>sizeSmallerSpin</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>412</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
